.github/workflows/build_test.yml

```
@@ -1,46 +1,74 @@
+    needs: choose-runner
+    if: ${{ (github.event_name == 'pull_request' && contains(join(github.event.pull_request.labels.*.name, ','), 'ci-runner:')) || (github.event_name == 'workflow_dispatch' && inputs.runner_label != '') }}
+    runs-on: ${{ fromJSON(needs.choose-runner.outputs.runs_on_json) }}
     timeout-minutes: 120
     steps:
       - name: checkout
         uses: actions/checkout@v1
         with:
           submodules: recursive
+
       - name: run fmt check
         uses: actions-rs/cargo@v1
```

actionlint (1.7.7)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)